packages/orbit-components/src/Card/CardSection/types.d.ts

```diff
@@ -6,19 +6,30 @@ import type * as React from "react";
 import type * as Common from "../../common/types";
 import type { As } from "../../Heading/types";
 
-export interface Props extends Common.Globals {
+type ExpandableConditionalProps =
```

Comment:
Could you try to write a type abstraction for this? It's kinda becoming a pattern for our components, so it would be nice to have that

Reply:
I am not sure I understand your request. You want a generic conditional type?